### DISCUSSION TITLE
Mysql persistence

### DISCUSSION THREAD

**Discussion by spasaria** [Netflix/conductor-community#69] [Q&A] [answered]
When i link mysql instance to conductor (by passing in the config map the url of mysql table for the conductor server v 3.8.3) i receive this error:
![errorConductor](https://user-images.githubusercontent.com/36420508/175058226-f93eb376-7422-44dd-85d5-b91ca7c1c1a0.PNG)

`***************************
APPLICATION FAILED TO START
***************************

Description:

Parameter 2 of constructor in com.netflix.conductor.core.execution.DeciderService required a bean of type 'com.netflix.conductor.dao.MetadataDAO' that could not be found.

Action:

Consider defining a bean of type 'com.netflix.conductor.dao.MetadataDAO' in your configuration.` 

Can someone help me to configure new conductor server with mysql persistence?
Thank you.

**Comment by v1r3n** [comment, marked as answer]
Hi @spasaria you need to use the conductor-community build that adds the mysql dependency:

https://github.com/Netflix/conductor-community/tree/main/community-server

**Reply by spasaria** [reply]
Hi @v1r3n thanks for the response. I continue to have problem in the installation process.
In particular i don't understand the phase:

"How can I use this binary in my docker builds
Follow the build process for docker at the [main repo](https://github.com/Netflix/conductor/tree/main/docker) and replace the conductor server with the output from this repository."

Where i can find a more clear installation process that describe all the steps?

Thank you.

**Reply by azriel46d** [reply]
@spasaria Had the same issue and the documentation is not that clear. 
To achieve the above I copied and adjusted dockerFile from the conductor repo , added full clones of both repositories and then replaced the build file.
Here is my dockerFile, hope its understanadable
``` 
#
# conductor:server - Netflix conductor server
#
# ===========================================================================================================
# 0. Builder stage
# ===========================================================================================================
FROM openjdk:11-jdk AS builder

LABEL maintainer="Netflix OSS <conductor@netflix.com>"

WORKDIR /

RUN git clone https://github.com/Netflix/conductor.git
RUN git clone https://github.com/Netflix/conductor-community.git

WORKDIR /conductor-community

# Build the server on run
RUN ./gradlew build -x test --stacktrace



# ===========================================================================================================
# 1. Bin stage
# ===========================================================================================================
FROM openjdk:11-jre

LABEL maintainer="Netflix OSS <conductor@netflix.com>"

# Make app folders
RUN mkdir -p /app/config /app/logs /app/libs

# Copy the compiled output to new image
COPY --from=builder /conductor/docker/server/bin /app
COPY --from=builder /conductor/docker/server/config /app/config
COPY --from=builder /conductor-community/community-server/build/libs/conductor-community-server-*-boot.jar /app/libs/conductor-server-*-boot.jar

# Copy the files for the server into the app folders
RUN chmod +x /app/startup.sh

HEALTHCHECK --interval=60s --timeout=30s --retries=10 CMD curl -I -XGET http://localhost:8080/health || exit 1

CMD [ "/app/startup.sh" ]
ENTRYPOINT [ "/bin/sh"]
```

**Comment by BhuhanJ** [comment]
Hi Community,

Anyone can help to integrate netflix conductor with mysql db without docker.
